### ISSUE TITLE
Some MP3's can't be submitted

### ISSUE THREAD

**Issue by gavspav** [roundware/roundware-web-template#12]
On my OSX system, some of my MP3's have icon's relating to the song - something to do with itunes and/or apple music? Metadata?
These mp3's - as well as appearing different, won't allow me to rename them.

For some reason I can't submit this type to the server.
The other files which have no icon work fine.

**Comment by hburgund** [comment]
I have actually noticed this just recently as well; not sure if it is new or coincidence. I just re-compressed the problematic files to mp3 which seems to have stripped out any of the metadata or other oddities that were messing things up. Clearly, this is not a real solution, but can help temporarily. I will investigate more.

**Comment by gavspav** [comment]
mp3's that don't upload do upload after I have removed all tags. I'm using this: [https://apps.apple.com/us/app/tag-editor-free/id984278082?mt=12](https://apps.apple.com/us/app/tag-editor-free/id984278082?mt=12&fbclid=IwAR1rlT5M1vPap7G--WTOd8l4nTCxbqkwSb8l54TJpwrAZ1aiIeeZdXSzu34)

**Comment by hburgund** [comment]
@shreyas-jadhav any way we could strip out mp3 tags prior to uploading files?

**Comment by shreyas-jadhav** [comment]
yes, i used this module to get rid of tags https://github.com/egoroof/browser-id3-writer
here is the commit https://github.com/shreyas-jadhav/roundware-web-template/commit/d3ca5dcfbff53142fa8076bcc40bf3f93a0f70a2

LMK if works now.

**Comment by hburgund** [comment]
seems to work for me, but if OP @gavspav can also check, that would confirm.

**Comment by gavspav** [comment]
Yeah this seems to work fine now!
